visualization/app/codeCharta/assets/scenarios.json

```diff
@@ -20,10 +20,10 @@
         "invertColorRange": false
       },
       "dynamicSettings": {
-        "areaMetric": "unary",
+        "areaMetric": "Average Complexity*",
         "heightMetric": "Average Complexity*",
         "colorMetric": "Average Complexity*",
-        "distributionMetric": "unary"
+        "distributionMetric": "Average Complexity*"
```

Comment:
We should get some feedback on how this scenario should be set now.

Reply:
This might be a blocker as far as I see it?

Reply:
I think that the areaMetric should be set to `rloc` all the time. In this case I'd replace the unary-metric with rloc here.